client/blocks/reader-subscription-list-item/style.scss

```diff
@@ -121,24 +124,29 @@
 
 	@include breakpoint( "<660px" ) {
 		flex-direction: row;
+		max-height: none;
```

Comment:
It was to just remove the original max-height set in L117 once it hits mobile, but we can do without it: https://github.com/Automattic/wp-calypso/pull/15221/commits/1a7fed82bd930b80dceb9bd6d5b25fc1dc82eecf